### ISSUE TITLE
When the keywords is empty,  the effect is unexcepted

### ISSUE THREAD

**Issue by ccfish86** [TonPC64/vue-highlight-text#5]
`vue-highlight-text` is very useful for me.
but when we use it with v-html,such as code 'v-highlight="{keyword: keywords, overWriteStyle: {color: '#00C100'}}" v-html="articlesInfo"'. when the keywords is empty string or array contains empty element, the content is alse be replaced as image follows, then ,  and when the keywords is null/undifined, then content is display as blank.

![image](https://user-images.githubusercontent.com/15622514/43435850-683b79d2-94b4-11e8-91b4-09fd3ef2cf3f.png)

**Comment by ccfish86** [comment]
Now,  it's works when the empty/blank is skiped. 
example:
```
const highlightSearch = function(message, keyword, flags, styleString) {
  let newKeyword = keyword
  let regexWord = ''
  if (typeof keyword === String) {
    if (/^\s*$/.test(keyword)) {
      return message;
    }
    regexWord =  escapeRegExp(newKeyword)
  } else if (Array.isArray(keyword) && keyword.length > 0) {
    let kws = keyword.filter(s => !/^\s*$/.test(keyword));
    if (kws.length === 0) {
      return message;
    }
    regexWord = kws.map(k => escapeRegExp(k)).join('|')
  } else {
    // console.warn('type is not String or Array')
    // return ''
    return message
  }

  const match = new RegExp(`(${regexWord})`, flags)
  if (match.test(message)) {
    return message.replace(match, `<span ${styleString}>\$&</span>`)
  }
  return message
}
```

**Comment by TonPC64** [comment]
can you pull request?

**Comment by TonPC64** [comment]
merged and update to version 1.5.0
